OsmAnd/src/net/osmand/plus/mapcontextmenu/controllers/WptPtMenuController.java

```diff
@@ -102,6 +106,11 @@ public void buttonPressed() {
 				}
 			}
 		}
+
+		TransportStop transportStop = builder.findUpdatedTransportStop(wpt.getExtensionsToRead(), wpt.getAmenityOriginName(), wpt.getTransportStopOriginName(), wpt.lat, wpt.lon);
```

Comment:
We don't need to display Transport Stop on saved Favorites / POI (1) it never worked 2) it will simplify the code)